src/main/Config.cpp

```diff
@@ -305,6 +305,10 @@ Config::Config() : NODE_SEED(SecretKey::random())
     EMIT_LEDGER_CLOSE_META_EXT_V1 = false;
 
     FORCE_OLD_STYLE_LEADER_ELECTION = false;
+    // This is not configurable for now. It doesn't need to be a network-wide
+    // setting, but on the other hand there aren't many good values for it and
+    // it's not clear what the right way to configure it would be, if at all.
+    SOROBAN_PHASE_STAGE_COUNT = 1;
```

Comment:
I think you've mentioned this setting should be dynamically determined by the contents of the tx queue instead of hardcoded. What's the plan wrt that?

Reply:
I think that would be a separate PR. But I think we could make a few tx sets in parallel and use the one with the smallest amount of stages that provides roughly the same utilization.